levels/level1.csv

```diff
@@ -0,0 +1,4 @@
+# delta,spawn_node,enemy
```

Comment:
I wonder if there's a merit to defining these as CSV files rather than scenes? 
Similar to how the projectile effects use a shared base, a level could simply be a custom scene type that exports these variables

It would be "larger" than a csv, but from what I've seen tscn files aren't large enough for that to really be a problem unless we're talking tens of thousands

Reply:
My thinking is that to program a full level in the game editor is going to require a huge number of nodes, i.e. create every timer for every spawn event in that level?
My idea with the csv is that we instead just encode what we want those timers to be, and then on level load we create a scene from that data.
Keen to hear alternative solutions to the problem though if you have some 👍

Reply:
I was thinking that the timers etc. would be programattically instantiated the same way as via CSV,

Really the only difference in my suggestion is that the data that's in a CSV would be stored as a scene extending some base scene, similar to how I did projectile effects.

That base scene would export variables that can be set in code or the godot UI -- each of those variables would be a CSV column.

So the boilerplate is all in place, we just define the CSV data structure as a scene and input those values in godot directly, instead of a separate file.

Reply:
I don't know if that's stricltly "better" -- the csv is probably more manageable at a scale, and creates an adhoc entrypoint so we could load whatever we want in

The "every level is its own pre-defined scene" is more manageable for individual complex scenes (and allows us to extend / script levels), but requires more effort for each level than the CSV approach